wit/mqtt.wit

```diff
@@ -0,0 +1,32 @@
+interface mqtt {
+  /// Errors related to interacting with Mqtt
+  variant error {
+      /// An invalid address string
+      invalid-address,
+      /// There are too many open connections
+      too-many-connections,
+      /// A retrieved value was not of the correct type
+      type-error,
```

Comment:
I'm probably missing something but I don't see where this gets used?

Reply:
Will remove, not used indeed.